site-src/geps/gep-1426.md

```diff
@@ -146,6 +141,23 @@ This should be enforced by controller validation, which should set an `Accepted:
 
 The logic described here is a bit verbose, but could be validated with conformance tests, and is intended to allow an `HTTPRoute` configuring mesh traffic rules to additionally specify a `Gateway` `parentRef` (or `HTTPRoute` `parentRef` if [GEP-1058: Route Inclusion and Delegation](https://github.com/kubernetes-sigs/gateway-api/pull/1085) moves forward) to apply the same routing rules when exposing a mesh service outside the cluster.
 
+### Namespace boundaries
+
+In a mesh, routes can be configured by two personas:
+
+* Service producers, who want to modify behavior of *inbound* requests to their services
+* Service consumers, who want to modify behavior of *outbound* requests to other services.
+
+While these concepts are not directly exposed in the API, a route is implicitly fulfilling one of these rolls and behaves differently depending on the role.
+
+A route is a producer route when the `parentRef` refers to a service in the *same namespace*. This route SHOULD apply to all incoming requests to the service, including from clients in other namespaces.
+
+Note: Some implementations may only be able to apply routes on client-side proxies. As a result, these will likely only apply to requests from clients who are also in the mesh.
+
+A route is a consumer route when the `parentRef` refers to a service in *another namespace*. Unlike producer routes, consumer routes are scoped only the same namespace. This ensures that for traffic between two namespaces, another unrelated namespace cannot modify their traffic.
+
+Routes of either type can send traffic to `backendRefs` in any namespace. Unlike `Gateway` bound routes, this is allowed without a `ReferenceGrant`. In `Gateway`, routes are opt-in; by default, no Services are exposed (often to the public internet), and a service producer must explicitly opt-in by creating a route themselves, or allowing another namespace to via `ReferenceGrant`. For mesh, routes *augment* existing Services, rather than exposing them to a broader scope. As a result, a `ReferenceGrant` is not required. Access control, if desired, is handled by authorization policies.
```

Comment:
This paragraph is confusing to me. I think the "In Gateway" sentence is intended to refer to north-south traffic coming from a Gateway, but the current wording is a bit confusing to me. 

Is it always true that a mesh implementation is not going to be subject to a confused deputy attack? It seems like at least some mesh implementations may be able to route traffic to locations that would otherwise be inaccessible, and thus ReferenceGrant would still be an important mechanism.

When you reference "authorization policies", I think it's important to state that those are not currently defined by Gateway API and are out of scope for this GEP.

Reply:
In Gateway the guidance is basically: "You need Grant unless you have some exceptional implementation that handles it otherwise". I'll change the wording here to "You *don't* need Grant, unless you have some exceptional implementation that *doesn't* handle it otherwise".

Essentially the same but most/all meshes will not needed it, so its just flipping the default